src/config/ConfigManager.hpp

```diff
@@ -49,6 +49,7 @@ struct SWorkspaceRule {
     std::optional<int>     rounding;
     std::optional<int>     decorate;
     std::optional<int>     shadow;
+    std::string            onCreatedEmptyRunCmd;
```

Comment:
why not make it an `std::optional` too? Keeps consistency.

Reply:
Addressed in 0f0711f